### PR TITLE
粗校，稍微改了改个别符号，语序

### PR DESCRIPTION
**Pull request by LiquorACE**

### CODE REVIEW

File: text/GCCC1AU.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@
 0x06800F35=~z~真的，男人都爱我。我从你看我时候的样子里就能看出来。
 
 ;0x06A9FF99=~z~They got a reality TV show for everything now - rich people, cars, hookers.
-0x06A9FF99=~z~现在什么题材的真人秀都有——富人、汽车、妓女。
+0x06A9FF99=~z~现在什么题材的真人秀都有——富人们、汽车、妓女们。
```
Comment (BlueNoBaka) [inline text/GCCC1AU.txt:30]: 中文复数没必要加"们" "富人"和"妓女"本身就可以代指一类人
别被英文的逻辑套住

File: text/GCCC1AU.txt [hunk 2]
```diff
@@ -51,7 +51,7 @@
 0x0BF5ADD7=~z~看什么？
 
 ;0x0ED6C7EE=~z~Don't you make a difference to the people you look after - you're a nurse?
-0x0ED6C7EE=~z~你不是已经对你看护的人做出影响了吗，你可是护士啊？
+0x0ED6C7EE=~z~你不会对你的病人们也区别对待吧，你还是不是护士啊？
```
Comment (BlueNoBaka) [inline text/GCCC1AU.txt:54]: Carmen: I want to do something important, really make a difference, you know?
Niko: Don't you make a difference to the people you look after - you're a nurse?
"make a difference"就没区别对待的意思

File: text/GCCC1AU.txt [hunk 3]
```diff
@@ -105,7 +105,7 @@
 0x21F49317=GCC2_EXCLUSIVE_GD
 
 ;0x24577307=~z~You want to read stuff out in front of a camera? What's important about that?
-0x24577307=~z~你想在摄影机前念广告？那能有什么价值？
+0x24577307=~z~你想在摄影机前念那玩意？那能有什么价值？
```
Comment (BlueNoBaka) [inline text/GCCC1AU.txt:108]: (接在上面Niko说"Don't you make a difference to the people you look after - you're a nurse?"后)
Carmen: Screw that, honey. I want people to notice me. I want to be a television personality.
Carmen: Host a TV show like that one - I'm Rich.
Niko: You want to read stuff out in front of a camera? What's important about that?
"stuff"应该是指主持人需要念的台词 直接写"广告"确实不妥 不过写"那玩意"就是意义不明了

File: text/GCCC2AU.txt [hunk 4]
```diff
@@ -465,7 +465,7 @@
 0xD2CC16B0=~z~我喜欢和你出来逛，Niko，但我有个问题。是个更加深入的话题。
 
 ;0xD41DDD60=~z~Surely Tony McTony is a man?
-0xD41DDD60=~z~Tony McTony应该是男的吧？
+0xD41DDD60=~z~你确定Tony McTony是男的？
```
Comment (BlueNoBaka) [inline text/GCCC2AU.txt:468]: Carmen做了指甲给Niko看 说起给她做指甲的朋友还给不少名人做过指甲 然后列举了两个人名
Carmen: Yes. Jill van Crastenburg was just in before me. Oh, and Tony McTony.
Niko: Surely Tony McTony is a man? (对男人做指甲感到惊讶)
Carmen: Yes - but there is nothing feminine about looking after yourself.

File: text/GCCC1AU.txt (hunk 3 above)
Comment (LiquorACE) [reply text/GCCC1AU.txt:108]: > 长对话在文本里是乱序的 有些地方只看单句改就会前言不搭后语，我翻的时候会按正常的对话顺序翻 所以有些地方会跟原文直译有出入(包括这次提交的GCCC1AU、GCCC2AU和GCCDCAU)
> GCKC1AU、GCKC2AU和GCKDCAU这三个你要是还没开始校对就别校对了，就算要校对也别做除了改错字之外的改动，这两个文本是与Kate约会时的长对话。
> PI开头的路人任务的文本也是乱序长对话 我按正常顺序翻的 应该不会有错译，你要校对的话找下错字就行了 连标点都别改一个。
> Kiki约会的长对话文本应该没按对话顺序翻译 所以你可以放心地大刀阔斧改

原来如此…那剩下的除了KIKI我就只看看有无错字标点吧

Comment (LiquorACE) [reply text/GCCC1AU.txt:108]: 上面你说的错误我都改回来了，还需要再上传吗

Comment (BlueNoBaka) [reply text/GCCC1AU.txt:108]: 如果你是在本地改的就需要，用github的edit file功能改的就不需要
这次的答案应该是需要

**Pull request merged**